examples/todo-intermediate/src/models/todo-list.model.ts

```diff
@@ -4,7 +4,10 @@ import {Todo} from './todo.model';
 @model()
 export class TodoList extends Entity {
   @property({id: true})
-  id: number;
-  @property() createdAt: Date;
-  @hasMany(Todo) todos: Todo[];
+  id?: number;
+  @property({required: true})
+  title: string;
+  @property({required: true})
+  lastModified: string;
```

Comment:
Hmm, do we have an easy way how to enforce `lastModified` to be always accurately set? It must not be a responsibility of clients to keep this field updated whenever any data changes!

I don't think juggler supports such feature out of the box, one needs to use `before save` or `persist` hook, which is not documented for LB4 models/repositories (yet).

I am proposing to remove this property. If we need another property besides id and title, then I am proposing to define a property called `color` (inspired by Apple's Reminders app).